integrationExamples/gpt/esp_example.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <link rel="icon" type="image/png" href="/favicon.png">
     <script async src="//www.googletagservices.com/tag/js/gpt.js"></script>
-    <script async src="prebid.js"></script>
+    <script  type="text/javascript"  src="../../build/dev/prebid.js" async></script>
```

Comment:
What's the reason for this change?

Reply:
![image](https://github.com/prebid/Prebid.js/assets/116539741/19f3ceb2-41eb-49f4-b6de-6f5ee99ac6bd)
This is the error I got when I trying to test esp_example, and I found this file hasn't been touch for a while now, so i am trying to fix the example